### DISCUSSION TITLE
Limit number of addresses that can be added to user

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#12390] [Ideas] [answered]
While it makes sense that webshops clients might have multiple addresses defined, sometimes you might only need one. 
For example if you are just building some kind of portal and not webshop. And ability to define multiple addresses per user is not needed.

**Comment by brandonkelly** [comment, marked as answer]
Craft Commerce sites generally need at least two (your primary billing and shipping addresses) so I don’t think we’d want to give admins the ability to artificially limit this.
